### PR TITLE
chore(macros): update MDN Sidebar strings for zh-CN

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
related to https://github.com/mdn/translated-content/issues/11063
 
cc: @yin1999 @jasonren0403

### CODE REVIEW

File: kumascript/macros/MDNSidebar.ejs [hunk 1]
```diff
@@ -160,26 +160,26 @@ const text = mdn.localStringMap({
...
+      "Learn": "学习论坛",
+      "Issues": "Issue 指南",
+      "PRs": "Pull request 指南",
+      "Content": "content 仓库相关指南",
+      "Users": "角色和团队",
+    "Writing_guide": "写作相关指南",
+      "Write_overview": "概览",
+      "What_write": "MDN 收录规则",
+      "Style_guide": "写作风格指南",
+      "Howto": "MDN 写作指南",
```
Comment (yin1999) [inline kumascript/macros/MDNSidebar.ejs:179]: I would like '操作指南', @jasonren0403 any idea?

```suggestion
      "Howto": "操作指南",
```

File: kumascript/macros/MDNSidebar.ejs [hunk 2]
```diff
@@ -160,26 +160,26 @@ const text = mdn.localStringMap({
...
+    "Community_guide": "社区指南",
+      "Comm_overview": "概览",
+      "Contributing": "为 MDN 做贡献",
+      "Open_source": "开源规约",
+      "Communication": "交流频道",
+      "Discuss": "讨论",
+      "Learn": "学习论坛",
+      "Issues": "Issue 指南",
+      "PRs": "Pull request 指南",
+      "Content": "content 仓库相关指南",
```
Comment (yin1999) [inline kumascript/macros/MDNSidebar.ejs:173]:
```suggestion
      "Content": "content 仓库指南",
```

File: kumascript/macros/MDNSidebar.ejs (hunk 1 above)
Comment (awxiaoxian2020) [reply kumascript/macros/MDNSidebar.ejs:179]: 或者“写作实践指南”？

Comment (jasonren0403) [reply kumascript/macros/MDNSidebar.ejs:179]: "写作指南"(maybe prefixed by 'MDN') may be better, as most content here tells us 'how to document', 'how to write', and some other methods to prepare writing the document.

Comment (yin1999) [reply kumascript/macros/MDNSidebar.ejs:179]: Okay, let's keep it as "写作指南", as it's now not duplicated with other content above.

**Review by yin1999 — APPROVED** [review]
Looks good to me, thanks @awxiaoxian2020 🎉

**Review by jasonren0403 — APPROVED** [review]
👍

**Comment by awxiaoxian2020** [comment]
ping @LeoMcA to review it :)

**Review by LeoMcA — APPROVED** [review]
lgtm, thanks :)

**Pull request merged**